DuckDuckGo/dist/metadata.json

```diff
@@ -8,10 +8,96 @@
     "en": "The Internet privacy company that empowers you to seamlessly take control of your personal information online, without any tradeoffs."
   },
   "service": "DuckDuckGo",
-  "version": "1.1.1",
+  "version": "1.2.0",
   "logo": "https://i.imgur.com/zl3jobl.png",
   "thumbnail": "https://i.imgur.com/VlPQGur.png",
   "color": "#FF5900",
   "tags": ["internet", "browing", "search"],
-  "category": "other"
+  "category": "other",
+  "settings": [
+    {
+        "id": "homepageMessage",
+        "title": "Homepage Message",
+        "icon": "fas fa-home",
+        "value": "Chilling on the homepage",
+        "placeholder": "Chilling on the homepage"
```

Comment:
Make it more generel as default like, Viewing the homepage, or Browsing the homepage